packages/sync-actions/test/customer-sync.spec.js

```diff
@@ -417,4 +421,20 @@ describe('Actions', () => {
     ]
     expect(actual).toEqual(expected)
   })
+
+  test('should build `setStores` action', () => {
+    const before = {
+      stores: ['canada'],
+    }
+    const now = {
+      stores: ['canada', 'usa'],
+    }
+    const actual = customerSync.buildActions(now, before)
+    expect(actual).toEqual([
+      {
+        action: 'setStores',
+        stores: ['canada', 'usa'],
```

Comment:
Are you really sure this works against our api? according to our docs (https://docs.commercetools.com/http-api-projects-customers#set-stores-beta) you have to provide a `ResourceIdentifier` (https://docs.commercetools.com/http-api-types.html#resourceidentifier) and not just a `string`...

Reply:
2403522